azure-communication-ui/chat/build.gradle

```diff
@@ -47,13 +47,22 @@ dependencies {
     implementation "androidx.fragment:fragment-ktx:$androidx_fragment_ktx_version"
     implementation "androidx.navigation:navigation-fragment-ktx:$androidx_navigation_fragment_ktx_version"
 
-    implementation "com.microsoft.fluentui:fluentui_others:$microsoft_fluent_ui_version_v2"
-    implementation "androidx.activity:activity-compose:$androidx_activity_compose_version"
-
     api ("com.azure.android:azure-communication-chat:$azure_chat_sdk_version")
     api ("com.azure.android:azure-communication-common:$azure_common_sdk_version")
 
+    implementation "com.microsoft.fluentui:fluentui_others:$microsoft_fluent_ui_version_v2"
+    implementation "androidx.compose.ui:ui:$compose_version"
+    implementation "androidx.compose.material:material:$compose_version"
+    implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
+    implementation "androidx.activity:activity-compose:$androidx_activity_compose_version"
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:$androidx_lifecycle_runtime_ktx_version"
+    implementation "androidx.compose.material3:material3:$compose_material3_version"
+    implementation "androidx.compose.material3:material3-window-size-class:$compose_material3_version"
```

Comment:
I don't think we need any material dependencies.

Reply:
Added for the custom topAppbarDeafaults. 

import androidx.compose.material.MaterialTheme
import androidx.compose.material.Text
import androidx.compose.material3.CenterAlignedTopAppBar
import androidx.compose.material3.TopAppBarDefaults

This is used in the topbar of the chatView. But I agree if we build this components ourself, we can remove such dependencies.

Reply:
Removed the material3 dependency.